src/app/about-us/about-us.component.html

```diff
@@ -1,13 +1,13 @@
 <h4 class="center-align">
-  ・{{ 'about-us.main-title' | translate }}・
+  ・{{ 'about-us.who.title' | translate | uppercase }}・
 </h4>
 <div class="parallax-container">
   <div appParallax class="parallax">
     <img src="assets/images/GP4Tech-Team.jpg" />
   </div>
 </div>
 <div class="container">
-  <br>
+  <br />
```

Comment:
We don't need to use this notation, this is for giving support to XHTML and we are not supporting this doctype. According to HTML5 standards, the notation is: ```<br>```